stix_shifter/stix_translation/src/modules/guardium/json/to_stix_map.json

```diff
@@ -0,0 +1,163 @@
+{
+  "Login Succeeded": {
```

Comment:
`number_observed` is a always required to be an attribute of the observed_data object. Will "Login Succeeded" always be present in the Guardium results?

Reply:
Agreed.  Fixed by using a transformer - SetToOne.